### PR TITLE
Issue #28 resolved

### PR DESCRIPTION
**Pull request by hxpdong**

### CODE REVIEW

File: todo_app/public/index.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/todoappfavicon.ico" />
```
Comment (ShehrozIrfan) [inline todo_app/public/index.html:5]: Hey @hxpdong !
You can rename `todoappfavicon.ico` to just `favicon.ico`

**Comment by ShehrozIrfan** [comment]
Hey @hxpdong !
Do add the screenshot of the site after the changes.
So, that we can see the output for `favicon`, how it looks.

And also, you added the binary file for `favicon`, is there any specific reason for this?

**Comment by hxpdong** [comment]
image

**Comment by hxpdong** [comment]
> image

That is the screenshot of application after I modified it. 
-I name the new icon is todoappfavicon.ico to distinguish with origin icon
-todoappfavicon.ico is the image which you provided. I just remove this background and change file extension to .ico. 
-I just tested with your image, which after I remove background with file extension is .png, the application still works successfully:
image

**Comment by hxpdong** [comment]
If you like, I will remove favicon.ico and add your icon in public folder, this icon will be named favicon.png.
After that, I will add new pull for you, If you like

**Pull request closed without merging**